sdk/eventhub/azure-eventhub/azure/eventhub/aio/_eventprocessor/event_processor.py

```diff
@@ -250,6 +245,7 @@ def _create_tasks_for_claimed_ownership(
                         self._consumer_group,
                         err,
                     )
```

Comment:
We're already logging a warning in `def _process_error` so this will log double warnings. Was that intended?

Reply:
should merge them into one